tools/IoTEdgeDevOps/DevOpsLibTest/BuildExtensionTest.cs

```diff
@@ -39,9 +43,9 @@ public void TestDisplayName()
             Assert.AreEqual("Edgelet CI", BuildDefinitionId.EdgeletCI.DisplayName());
             Assert.AreEqual("Edgelet Packages", BuildDefinitionId.EdgeletPackages.DisplayName());
             Assert.AreEqual("Edgelet Release", BuildDefinitionId.EdgeletRelease.DisplayName());
-            Assert.AreEqual("End-to-End Test", BuildDefinitionId.EndToEndTest.DisplayName());
+            Assert.AreEqual("New E2E Test", BuildDefinitionId.EndToEndTest.DisplayName());
```

Comment:
nit: Do we need to have the 'New' in the name?

Reply:
We used to differentiate between multiple E2E pipelines with this tag, but if we remove it, we would need to change many things.